dlfcn.c

```diff
@@ -312,9 +323,11 @@ int dlclose( void *handle )
     return (int) ret;
 }
 
+__declspec(noinline) /* Needed for _ReturnAddress() */

```

Comment:
This seems to be MSVC-specific attribute. Should be used ` __attribute__ ((noinline))` instead for GCC and Clang or this is not necessary for some reason?

Reply:
You are already using `__declspec(dllexport)` in dlfcn.h/dlfcn.c and also in tests. Even for GCC / Clang. So I chose `__declspec(noinline)` instead of `__attribute__ ((noinline))`.

I tested `__declspec(noinline)` with i686-w64-mingw32-gcc 6.3.0 and it is working fine. Seems that gcc supports some of MSVC `__declspec` attributes (dllexport/noinline).

Reply:
Great, thanks!